crates/core/migrations/2022-03-02-220455_add_store_addres_col_to_store_config_jsons_table/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table store_config_jsons
+add column store_address varchar(48) default ' ' not null;
```

Comment:
```suggestion
add column store_address varchar(48) default ' ' not null;
```
This will likely cause deserialization errors in the live db, can we make this nullable instead?